feat: GraphQL OTel bridge example app (#325)
newrelic/newrelic-node-examples: README.md

* [GraphQL Dataloader](./graphql-koa-dataloader) - example using Apollo Server, koa and GraphQL dataloader
* [Kafkajs](./kafkajs) - example demonstrating [kafkajs](https://kafka.js.org/) with the Node.js agent
* [Knex](./knex) - example demonstrating knex with the Node.js agent
* [Mock Infinite Tracing Server](./mock-infinite-tracing-server) - mock gRPC server to use to locally test [infinite tracing](https://docs.newrelic.com/docs/distributed-tracing/infinite-tracing/introduction-infinite-tracing/) with a Node.js applciation
* [Nest](./nestjs) - examples demonstrating NestJS with the Node.js agent
* [Next.js](./nextjs) - examples demonstrating Next.js with the Node.js agent
* [Prisma](./prisma-app) - example demonstrating Prisma with the Node.js agent
* [Sequelize](./sequelize-app) - example demonstrating sequelize with the Node.js agent
* [Express](./simple-express-app) - example demonstrating express with the Node.js agent
* [Source Maps](./source-maps) - typescript example using [--enable-source-maps](https://nodejs.org/dist/latest-v22.x/docs/api/cli.html#--enable-source-maps) to demonstrate proper stack traces the Node.js agent produces

## OpenTelemetry Bridge Examples

> When you put the Node.js agent in OpenTelemetry bridge mode, it can instrument all libraries OTel instruments. 

Reviewer: ```suggestion
When the Node.js agent is configured for OpenTelemetry bridge mode, it can instrument all libraries OTel instruments. 
```
